cni/pkg/config/config.go

```diff
@@ -126,22 +127,26 @@ func (c InstallConfig) String() string {
 	b.WriteString("MountedCNINetDir: " + c.MountedCNINetDir + "\n")
 	b.WriteString("CNIConfName: " + c.CNIConfName + "\n")
 	b.WriteString("ChainedCNIPlugin: " + fmt.Sprint(c.ChainedCNIPlugin) + "\n")
-	b.WriteString("CNINetworkConfigFile: " + c.CNINetworkConfigFile + "\n")
-	b.WriteString("CNINetworkConfig: " + c.CNINetworkConfig + "\n")
 
 	b.WriteString("LogLevel: " + c.LogLevel + "\n")
 	b.WriteString("KubeconfigFilename: " + c.KubeconfigFilename + "\n")
 	b.WriteString("KubeconfigMode: " + fmt.Sprintf("%#o", c.KubeconfigMode) + "\n")
 	b.WriteString("KubeCAFile: " + c.KubeCAFile + "\n")
 	b.WriteString("SkipTLSVerify: " + fmt.Sprint(c.SkipTLSVerify) + "\n")
 
+	b.WriteString("ExcludeNamespaces: " + fmt.Sprint(c.ExcludeNamespaces) + "\n")
```

Comment:
Is `K8sServiceAccountPath` omitted on purpose?

Reply:
Sorta. It's only override-able via unit tests so echoing it would just be noise.